syz-manager/manager.go

```diff
@@ -1164,7 +1165,12 @@ func saveReproAsset(dir string, asset *prog.ExtractedAsset) {
 	if asset.Error != nil {
 		err = osutil.WriteFile(path+".error", []byte(asset.Error.Error()))
 	} else if asset.Reader != nil {
-		err = osutil.WriteGzipStream(path+".gz", asset.Reader)
+		var f *os.File
+		f, err = os.Create(path + ".zz")
```

Comment:
How easy it is to decompress .zz on linux?
Potentially we can re-compress it to .gz here. This part is not performance-critical and user convenience is more important.
@a-nogikh what do you think?

Reply:
Yes, I think `.gz` is much more convenient here.

Note that the asset is also processed inside `uploadReproAsset`, where it (according to [`pkg/asset/type.go`](https://github.com/google/syzkaller/blob/f3cab5934d105412ced7a190300e5fb58d4ff425/pkg/asset/type.go#L58)) is then compressed using gzip. So we either need to extract the uncompressed version and leave all manager code as is, or we need to modify this part as well and check it all works fine before deploying it.